### PR TITLE
disable tensorflow/keras print-to-stdout with tf_disable_interactive_…

### PR DESCRIPTION
**Pull request by kba**
…logs, OCR-D/core#1091

### CODE REVIEW

**Comment by mikegerber** [comment]
(The code coverage result is weird)

File: ocrd_calamari/recognize.py [hunk 1]
```diff
@@ -18,12 +33,6 @@
...
         to_xml
 )
-from ocrd_utils import (
-        getLogger, concat_padded,
-        coordinates_for_segment, points_from_polygon, polygon_from_x0y0x1y1,
-        make_file_id, assert_file_grp_cardinality,
-        MIMETYPE_PAGE
-)
 
 from ocrd_calamari.config import OCRD_TOOL, TF_CPP_MIN_LOG_LEVEL
```
[the remark this answers is not in the document]
Comment (MehmedGIT) [reply ocrd_calamari/recognize.py:37]: Thanks for taking look and triggering my notifications on this. I overlooked the existence of this PR. Will check as well to see if there is a convenient way to resolve the issues.

[the remark this answers is not in the document]
Comment (mikegerber) [reply ocrd_calamari/recognize.py:37]: I'm also working on this :)

[the remark this answers is not in the document]
Comment (mikegerber) [reply ocrd_calamari/recognize.py]: This is fixed now, I pushed the fix to this PR

**Comment by mikegerber** [comment]
https://app.circleci.com/pipelines/github/OCR-D/ocrd_calamari/230/workflows/389c3843-ec1a-49fe-a252-4450458f9288/jobs/364

Broken for Python 3.6:
```
E   ImportError: cannot import name 'tf_disable_interactive_logs'
```

Python 3.6 is EOL since 2021-12-23, so it's time to say goodbye 😢

**Comment by MehmedGIT** [comment]
> Python 3.6 is EOL since 2021-12-23, so it's time to say goodbye

That version was dropped in core as well.

**Comment by mikegerber** [comment]
I removed testing on Python 3.6 in master and rebased this PR. This fixed it! 🚀

**Review by mikegerber — APPROVED** [review]
LGTM now

**Comment by mikegerber** [comment]
This required `ocrd >= 2.54` (for `tf_disable_interactive_logs`) and thus broke Python 3.6 (not that is to be supported anymore). I updated the requirement in f5b4187.

**Comment by mikegerber** [comment]
@MehmedGIT How would I (easily) test if this problem comes up again? Run the network/processing server?

I'm doing some code janitor work for #92 and this also involves some sorting of the import blocks. I disabled sorting for the block that disables interactive logs + imports TF + Calamari but I'd like to make sure.

**Comment by MehmedGIT** [comment]
@mikegerber, check here: https://pad.gwdg.de/Ty6IXzhIRa6AvDdC4kTy_g

**Pull request merged**